### ISSUE TITLE
[e2e] Photo uploaded in e2e room don't show off in room details / Files

### ISSUE THREAD

**Issue by jeangb** [vector-im/riot-android#737]
To reproduce:
1. Open or create a 1:1 room with encryption enabled
2. Send a photo from the device
3. Wait for the photo to be fully uploaded
4. Go into Room Details, and FILES tab

There is no photo.
Version 0.6.1-dev (e951ade)
Related to https://github.com/vector-im/vector-web/issues/2591
Related to https://github.com/vector-im/vector-ios/issues/819

**Comment by ylecollen** [comment]
i'm not sure we can fix this one.
There is no way to list the files on server side because they are encrypted.
The client uses server requests to list them.

We could only trigger a search in the known files (i.e downloaded ones).

**Comment by p0087** [comment]
Is this still the case? Was just about to submit a bug report myself before I found this. This is a pretty big deal breaker to convince people to switch from Whatsapp

**Comment by jeangb** [comment]
@p0087 No it's actually set up, I'm going to close it. 
FYI encrypted files that you can't decrypt (if you lost the keys for example) won't appear in this FILES tab.

**Comment by p0087** [comment]
Is it in the current 0.69 release? I've shared and received multiple files in the 0.69 release in an encrypted chat room and nothing appears.

**Comment by p0087** [comment]
No, these are photos I've uploaded days ago with no change to keys (no logging out). If I lost the keys I wouldn't be able to decrypt the messages either. Do I need to make another issue?

**Comment by jeangb** [comment]
Yes it sounds like an other bug

**Comment by p0087** [comment]
Actually on rereading OP's post, and the issues he linked (which are still open and marked active 19 days ago), I don't think it's a different issue...

**Comment by jeangb** [comment]
I'm sure this had been set up for Android at least. I don't reproduce it on android but I do on iOs.
We can reopen this issue, please details the step if there are different than the ones described in the first comment.
If you have ever see these files in local with the same device you use when you go into Details/FILES tab this is definitly the same problem.

**Comment by p0087** [comment]
Yes the steps were identical to the OP.

However just for fun, I just exported keys, logged out, logged in, imported keys, and was suddenly able to see all the media after 5 seconds waiting on a grey screen. As you said no luck on web/ios though.

**Comment by jeangb** [comment]
This is strange indeed that it works after importing the keys and not before...

**Comment by jeangb** [comment]
I tried to reproduce your bug with differents scenarios, but unsuccessfully. Maybe it happens but in the long term.
I'm closing the issue for the moment.
